testsuite/MDAnalysisTests/coordinates/test_coordinates.py

```diff
@@ -123,12 +123,12 @@ def test_frame_numbering(self):
 
     def test_frame(self):
         self.trajectory[0]
-        coord0 = self.universe.atoms.coordinates().copy()
+        coord0 = self.universe.atoms.positions.copy()
```

Comment:
Yeah it's always a fancy index so a copy

On Tue, 19 Apr 2016 00:06 David Dotson, <notifications@github.com> wrote:

> In testsuite/MDAnalysisTests/coordinates/test_coordinates.py
> <https://github.com/MDAnalysis/mdanalysis/pull/825#discussion_r60149867>:
>
> > @@ -123,12 +123,12 @@ def test_frame_numbering(self):
> >
> >      def test_frame(self):
> >          self.trajectory[0]
> > -        coord0 = self.universe.atoms.coordinates().copy()
> > +        coord0 = self.universe.atoms.positions.copy()
>
> Is it ever necessary to call .copy() on the array we return with
> ag.positions? Is it not always a copy?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/MDAnalysis/mdanalysis/pull/825/files/3f1f36d5c05f1026078a82a4a856462472c57500..330db1cd7ce9776aa3adbd7d63b5f9245cc53b0f#r60149867>
>